src/Core/Grid/Query/Filter/DoctrineFilterApplicator.php

```diff
@@ -80,8 +86,67 @@ public function apply(QueryBuilder $qb, SqlFilters $filters, array $filterValues
                         $qb->setParameter($name, sprintf('%s %s', $value['to'], '23:59:59'));
                     }
 
+                    break;
+                case SqlFilters::MIN_MAX:
+                    if (!isset($value['min_field']) && !isset($value['max_field'])) {
+                        break;
+                    }
+
+                    $minFieldSqlCondition = "$sqlField >= :{$filterName}_min";
+                    $maxFieldSqlCondition = "$sqlField <= :{$filterName}_max";
+
+                    switch ($this->computeMinMaxCase($value)) {
+                        case self::CASE_BOTH_FIELDS_EXIST_NOT_EQUAL:
+                            $qb->andWhere("$minFieldSqlCondition AND $maxFieldSqlCondition");
```

Comment:
Just a suggestion (totally optional) but you could use `sprintf` instead of double quotes to insert your variables I know it's not perfect on previous filter cases but we try to avoid using double quotes

However it's not blocking

Reply:
Why exactly double quotes are to be avoided? Because they consume more memory?

Because I would I agree about sprintf in most cases(as well as this specific one with AND in the middle), but in cases where you have `_min` I find `"$sqlField >= :{$filterName}_min";` much easier to read then `sprintf('%s >= :%s_min', $sqlField, $filterName);`

Reply:
Alright that makes sense. Applied changes.